apps/ledger-live-mobile/src/components/WebPlatformPlayer/WebViewV2.tsx

```diff
@@ -159,6 +163,22 @@ function useUiHook(): Partial<UiHook> {
   );
 }
 
+const useGetUserId = () => {
+  const [userId, setUserId] = useState("");
+
+  useEffect(() => {
+    let mounted = true;
+    getOrCreateUser().then(({ user }) => {
+      if (mounted) setUserId(user.id);
+    });
+    return () => {
+      mounted = false;
+    };
+  }, []);
```

Comment:
If I understand collectedly, `mounted` flag is to prevent setting `userId` when the component is already unmounted?

Reply:
Why this logic is different from desktop where simply calling `getEnv("USER_ID)` ?

Reply:
Because `USER_ID` is only set in LLD and not LLM

Reply:
Could you remind me of why the diff exists (only set in desktop)?

Reply:
I don't really know the why, just that it was not present and `getOrCreateUser` is used in every analytics file of LLM